ocdaction/challenges/views.py

```diff
@@ -27,23 +27,19 @@ def challenge_list(request):
         '-updated_at'
     )[:10]
 
-    for challenge in challenges_not_in_progress:
-        print(challenge.in_progress)
-
     challenges_in_progress_sorted = sorted(
         challenges_in_progress.all(),
         reverse=True,
         key=lambda c: int(c.get_latest_initial_anxiety_level())
     )
 
-    # challenge_in_progress = Challenge.objects.filter(in_progress=True)
-    anxiety_score_card = AnxietyScoreCard.objects.filter(challenge=challenges_in_progress).last()
+    challenge_in_progress = Challenge.objects.filter(in_progress=True)
+    anxiety_score_card = AnxietyScoreCard.objects.filter(challenge=challenge_in_progress).last()
 
     context = {
         'challenges_in_progess': challenges_in_progress_sorted,
         'challenges_not_in_progress': challenges_not_in_progress,
-        # 'another_challenge_in_progress': challenge_in_progress,
```

Comment:
Context changes
`        'challenge_in_progess': challenge_in_progress,`
and
`        'challenges_not_in_progress': challenges_not_in_progress_sorted,`

obviously then will need to make change to variable name in template